api/gateway/apimiddleware/process_field.go

```diff
@@ -26,6 +26,25 @@ func processField(s interface{}, processors []fieldProcessor) error {
 	t := reflect.TypeOf(s).Elem()
 	v := reflect.Indirect(reflect.ValueOf(s))
 
+	if t.Kind() == reflect.Struct {
+		err, done := processStructField(t, v, processors)
```

Comment:
The original code is moved to `processStructField` function.  
This refactoring is to solve **lint** complains.